core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/parser/antlr/ErrorThrowingListener.scala

```diff
@@ -29,8 +29,11 @@ class ErrorThrowingListener extends BaseErrorListener {
                            msg: String,
                            e: RecognitionException
   ): Unit = {
-    val errorMsg =
+    val errorMsg = if (msg.startsWith("mismatched input")) {
+      s"Error while parsing the expression. Offending symbol is on line $line at position $charPositionInLine. Error: $msg. For instance strings should be formatted as \"string\"."
```

Comment:
```suggestion
      s"Error while parsing the expression. Offending symbol is on line $line at position $charPositionInLine. Error: $msg. For example, strings should be formatted as \"string\"."
```

Reply:
Thanks! I am just experimenting for now and will likely remassage things with Beth :)